YapDatabase/Utilities/YapDatabaseCryptoUtils.m

```diff
@@ -407,10 +388,14 @@ + (nullable NSData *)deriveDatabaseKeyForPassword:(NSData *)passwordData saltDat
 {
     YapAssert(passwordData.length > 0);
     YapAssert(saltData.length == kSQLCipherSaltLength);
-
-    unsigned char *derivedKeyBytes = malloc((size_t)kSQLCipherDerivedKeyLength);
-    YapAssert(derivedKeyBytes);
-    // See: PBKDF2_ITER.
```

Comment:
Whoops, your branch is based on `master`.  It should be based on `release/unencryptedHeaders`.  Can you please rebase?  Sorry for the confusion.

Reply:
whoops. Done.